src/actions/jsmd-parser.js

```diff
@@ -0,0 +1,51 @@
+export function jsmdParser(fullJsmd) {
+  const jsmdLines = fullJsmd.split('\n')
+  const chunks = []
+  let currentChunkLines = []
+  let currentEvalType = ''
+  let evalFlags = []
+  let currentChunkStartLine = 0
+
+  for (const [i, line] of jsmdLines.entries()) {
+    if (line.slice(0, 2) === '%%') {
+      // if there is any match, a new chunk has started
+      // push the current chunk (unless it's on line 0), then reset
+      if (i !== 0) {
+        // DON'T push a chunk if we're only on line 0
+        chunks.push({
+          cellContent: currentChunkLines.join('\n'),
```

Comment:
We should be using the term `chunk` not `cell` here.

Reply:
fixed with commit  ff8094f